internal/resources/policy/kind/network/recipe/deny_all_to_pods_flatten_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package recipe
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	policyrecipenetworkmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/network"
+	policyrecipenetworkcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/network/common"
+)
+
+func TestFlattenDenyAllToPods(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		description string
+		input       *policyrecipenetworkmodel.V1alpha1CommonPolicySpecNetworkV1DenyAllToPods
+		expected    []interface{}
+	}{
+		{
+			description: "check for nil deny-all-to-pods recipe network policy",
+			input:       nil,
+			expected:    nil,
+		},
+		{
+			description: "normal scenario with valid deny-all-to-pods recipe network policy",
```

Comment:
Ensure that the unit tests written covers all the lines and if conditions

Reply:
<img width="1460" alt="Screenshot 2023-06-06 at 11 08 33 AM" src="https://github.com/vmware/terraform-provider-tanzu-mission-control/assets/32017029/fba35679-b669-4856-a1be-c142e296d374">